### DISCUSSION TITLE
İBAX PROJESİ GELECEGİN BİTCOİN HABERCİSİ

### DISCUSSION THREAD

**Discussion by Kingboss21** [IBAX-io/go-ibax#1199] [Ideas]
SAĞLAM BİR ALT YAPI VE PROJESİNE SAHİP BİR COİN HERKESİN YORUMLARINI BEKLİYORUM

**Comment by Skips1** [comment]
Hocam ne projeler geldi geçti ve söndü. Umarım İbax bunlardan biri olmaz ibaxı seviyoruz kanka

**Reply by Kingboss21** [reply]
AVAX " DA BÖYLEYDİ KANKA :dagger:

**Comment by sedat2855** [comment]
BELLİ OLMAZ

**Reply by Kingboss21** [reply]
bakalım herkese bol şans

**Comment by Pasa1944** [comment]
Neden olmasın proje gayet güzel

**Reply by Kingboss21** [reply]
katılıyorum .

**Comment by canavar23** [comment]
güzell

**Reply by Kingboss21** [reply]
alt yapısı saglam .

**Comment by goksancebeci** [comment]
Neden cüzdan bakiyesi ile airdrop sayfasındaki puan farklı

**Reply by Kingboss21** [reply]
onlar  demo amaçlı herşey puana baglı
